CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.4.2]
+### Changed
+- Remove `set-output` ([#100](https://github.com/MetaMask/action-create-release-pr/pull/100))
+- Bump actions ([#98](https://github.com/MetaMask/action-create-release-pr/pull/98))
```

Comment:
Nit: Maybe better to frame it as fixing the deprecation warnings, as that's the user-facing affect of these two changes.
```suggestion
- Resolve GitHub action deprecation warnings ([#98](https://github.com/MetaMask/action-create-release-pr/pull/98), [#100](https://github.com/MetaMask/action-create-release-pr/pull/100))
  - GitHub actions have been updated to v3, and the deprecated `set-output` command is no longer used.
```